automerge-example/webhooks/templates/automerge-tt.yaml

```diff
@@ -27,6 +22,14 @@ spec:
     description: push, pull_request
   - name: branch-name
     description: The branch name associated with push events
+  - name: commit-name
+    type: string
```

Comment:
Type support in a TriggerTemplate's params field was removed in Triggers 0.5 and was deprecated in older versions too (apparently it was never really supported, see ), can you remove all of the type: string references in here so we can work with Triggers 0.5 please? 

```
#532 removes the undocumented and unsupported type field from TriggerTemplate params. If you were using the field in your TriggerTemplate, you'll have to delete the resource, and recreate it after deleting the type field.
``` for some background, from https://github.com/tektoncd/triggers/releases/tag/v0.5.0.

I know you didn't add them all in the first place but we may as well while we're here. I checked for other TriggerTemplates and this is it - worth mentioning "this is known to work with Triggers 0.5" in the readme too I think.

I'm going to go ahead and test with it omitted anyway so we can be confident it's good.

Reply:
i've removed type (not tested on 0.5)